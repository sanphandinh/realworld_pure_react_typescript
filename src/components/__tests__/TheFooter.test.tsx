import React from 'react';
import renderer from 'react-test-renderer';
import TheFooter from '../TheFooter';

test('The footer render correctyly', () => {
  const tree = renderer.create(<TheFooter />).toJSON();
  expect(tree).toMatchSnapshot();
});
