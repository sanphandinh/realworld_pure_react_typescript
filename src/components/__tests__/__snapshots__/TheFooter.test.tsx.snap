// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The footer render correctyly 1`] = `
<footer>
  <div
    className="container"
  >
    <a
      className="logo-font"
      href="/"
    >
      conduit
    </a>
    <span
      className="attribution"
    >
      An interactive learning project from
       
      <a
        href="https://thinkster.io"
      >
        Thinkster
      </a>
      . Code & design licensed under MIT.
    </span>
  </div>
</footer>
`;
